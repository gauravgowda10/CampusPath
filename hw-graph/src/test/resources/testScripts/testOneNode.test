# Tests that a node can be added.

# Create a graph
CreateGraph oneNodeGraph

# Add a node
AddNode oneNodeGraph head

# Print the nodes in the graph
ListNodes oneNodeGraph
ListChildren oneNodeGraph head