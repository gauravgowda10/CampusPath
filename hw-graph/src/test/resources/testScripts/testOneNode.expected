# Tests that a node can be added.

# Create a graph
created graph oneNodeGraph

# Add a node
added node head to oneNodeGraph

# Print the nodes in the graph
oneNodeGraph contains: head
the children of head in oneNodeGraph are: